pages/account.tsx

```diff
@@ -24,6 +24,57 @@ import DataTable from 'components/Datatable';
 
 /* Types */
 import { Address } from '../interfaces';
+import { ethers, Wallet } from 'ethers';
+
+/* Config */
+import { RPC_URL } from '../config/RPC';
```

Comment:
I kept getting errors when trying to use `env`. Mainly it wasn't recognized.

Reply:
I would set it, and it would always be undefined in app

Reply:
To use `env` on client-side you need to prefix them with `NEXT_PUBLIC_*`

https://nextjs.org/docs/basic-features/environment-variables

Reply:
Otherwise, I can verify and reverify with no issues.

Reply:
> Otherwise, I can verify and reverify with no issues.

Can you confirm if your address is removed from db before you verify?

Reply:
Actually this might be something we could fix on the BE too, I think saving it before verification is actually unnecessary, but want to make sure we're on the same page

Reply:
Right now on verify BE will fetch eth account from DB but when I test on this branch no record is created (only the record in the nonce collection)

Reply:
> saving it before verification is actually unnecessary

I think there is a nonce needed for SIWE, so some storage is required temporarily. Can maybe store it on the Governator User item